opni-insights-service/app/main.py

```diff
@@ -267,14 +272,75 @@ def get_namespace_breakdown(start_ts, end_ts):
     return namespace_breakdown_dict
 
 
+async def get_overall_breakdown(start_ts, end_ts):
+    # Get the overall breakdown of normal, suspicious and anomalous logs within start_ts and end_ts.
+    overall_breakdown_dict = {"Normal": 0, "Suspicious": 0, "Anomaly": 0}
+    for anomaly_level in overall_breakdown_dict:
+        query_body = {
+            "query": {
+                "bool": {
+                    "must": [
+                        {"match": {"anomaly_level": anomaly_level}},
+                    ],
+                    "filter": [
+                        {"range": {"timestamp": {"gte": start_ts, "lte": end_ts}}}
+                    ],
+                }
+            }
+        }
+        try:
+            overall_breakdown_dict[anomaly_level] = (
+                await es_instance.count(index="logs", body=query_body)
+            )["count"]
+        except Exception as e:
+            logging.error(f"Unable to access Elasticsearch data. {e}")
+            return overall_breakdown_dict
+    return overall_breakdown_dict
```

Comment:
This is fine for now, but we should be using an aggregation in ES rather than a loop with multiple queries.